CONTRIBUTING.md

```diff
@@ -15,79 +15,78 @@ highly appreciated!
   > free to use any DVC project that you have available.
 
-## The demo project
+[extension development host]:
+  https://code.visualstudio.com/api/working-with-extensions/testing-extension
 
-The [demo project](demo) was initially forked from
-[dvc-checkpoints-mnist](https://github.com/iterative/dvc-checkpoints-mnist/tree/make_checkpoint),
-and is provided as a lightweight, convenient testbed to try the extension out
-with. It is not an exhaustive showcase of DVC's features, so testers are
-encouraged to try other DVC repositories -- especially real-world cases!
+## The demo project
 
-- Go to the `demo/` directory of this repo.
+The [demo project](demo) is provided as a lightweight, convenient testbed to try
+your changes to this extension.
 
-- It is recommended that you create the project's virtual environment with
-  Python 3.8 as some dependencies are not yet available for Python 3.9
+> **Note**: It is not an exhaustive showcase of DVC's features. Testers are
+> encouraged to try other DVC repositories -- especially real-world cases!
 
-- `yarn install` is a prerequisite for automatically setting up the demo
-  project.
+- Run `yarn setup:venv` from the _Terminal_ to set up a Python virtual
+  environment for the demo project (in `demo/.env`).
 
-- Run `yarn setup:venv` from the monorepo root (not `./demo`) to automatically
-  setup the Python virtual environment in the demo project.
+- Open the `./demo` project in a VS Code window, for example in the _Extension
+  Development Host_ (see previous section).
 
-- Once the environment has been setup you should navigate into the demo folder
-  activate the virtual environment and run `dvc pull`.
+- Pull the project data using this extension (either from the **DVC Tracked**
+  panel in the _File Explorer_ or the **DVC panel** in _Source Control_) or by
+  running `dvc pull` from _Terminal_.
 
-- In order to run experiments in our demo project we require the Python virtual
-  environment to be loaded.
+- In order to [run experiments] in the demo project, the [Python extension]
+  should be installed so that the virtual env is activated.
 
-- This will happen automatically IF the `ms-python.python` extension is
-  installed within the VS Code instance that you are developing against.
+[python extension]:
+  https://marketplace.visualstudio.com/items?itemName=ms-python.python
+[run experiments]:
+  https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/run-experiments.md
 
 <a id='warning'></a>
 
-> **Warning**  
-> When using any project that relies on an isolated Python environment (`conda`,
-> `venv`, etc.), Microsoft's
-> [Python extension](https://github.com/Microsoft/vscode-python) is required.
+> **Warning**: When using any project that relies on an isolated Python
+> environment (`conda`, `venv`, etc.), Microsoft's Python extension is required.
 > It's used by this extension to locate and utilize the required environment.
```

Comment:
I feel like this note should be moved out to the README or some other place since it can apply to any DVC project and not just to the demo/ project or contrib guidelines. Right?

Reply:
Agreed, I would include it in the Development environment section or at least closer to the top of this one. I think it is ok to duplicate